ODIN_II/regression_test/benchmark/task/syntax/simulation_result.json

```diff
@@ -2312,6 +2300,33 @@
         "Estimated LUTs": 1020,
         "Total Node": 1315
     },
+    "syntax/multi_assignment/k6_frac_N10_frac_chain_mem32K_40nm": {
```

Comment:
These are mine, but they fail simulation at the moment (synthesis should be fine though). Should I leave the failing vectors, or should I remove them and come back in future once 4 valued simulation is implemented

Reply:
No we keep failing tests in, (if you look around you'll see a few, and they pop up during verification (sig abort etc...) A lot of tests have been added even tho Odin doesn't support them, it's really just to speed up the development since the test are already in, easier to figure if you fixed an issue or not